pkg/nsx/services/inventory/compare.go

```diff
@@ -129,3 +131,20 @@ func compareContainerApplication(pre interface{}, cur interface{}, property *map
 		updateProperties["origin_properties"] = cur.(containerinventory.ContainerApplication).OriginProperties
 	}
 }
+
+func compareContainerClusterNode(pre interface{}, cur interface{}, property *map[string]interface{}) {
```

Comment:
Shall we also check origin_properties for Node?

Reply:
ok, added